site/content/how-to/control-plane-configuration.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: "Control Plane Configuration"
-description: "Learn how to dynamically update the Gateway Fabric control plane configuration."
-weight: 100
+title: "Control plane configuration"
```

Comment:
I don't think they need to have their own separate directories: in this particular example, it just seemed like an unnecessary step to have a single document in a standalone folder.

The bigger question is probably how to separate reference and how-to information since they have different levels of depth.